templates/mjml/forgotEmail.mjml

```diff
@@ -0,0 +1,26 @@
+<!-- see https://documentation.mjml.io for documentation -->
+<mjml>
+    <mj-body background-color="#ffffff" width="850px">
+        <mj-section background-color="#ffffff" padding-bottom="0px" padding-top="0px">
+            <mj-column vertical-align="top" width="100%">
+                <mj-image src="https://github.com/ystv/public-files/blob/master/background.png?raw=true" alt="" align="center" border="none" width="850px" padding="15px 0 15px 0"></mj-image>
+            </mj-column>
+        </mj-section>
+        <mj-section padding-bottom="0px" padding-top="0">
+            <mj-column vertical-align="top" width="100%">
+                <mj-text align="left" color="#363636" font-size="48px" font-weight="600" font-family="open Sans Helvetica, Arial, sans-serif" padding-left="25px" padding-right="25px" padding-bottom="0px" padding-top="50px">YSTV Security</mj-text>
```

Comment:
Should this be `"Open Sans", Helvetica, Arial, sans-serif`?

Reply:
Yes, in the event that one it not recognised by the browser then the others are fallback, that is my understanding
https://www.w3schools.com/cssref/pr_font_font-family.php

Reply:
You currently have `open Sans Helvetica` as one entry though

Reply:
Not sure about that, I think I designed it ages ago with the help of a different template and copied parts over, oh well, fixed now